utils/create-keying-sheet-tfm-facilities/index.ts

```diff
@@ -0,0 +1,38 @@
+import { WithoutId } from 'mongodb';
+import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
+import { Facility, TfmFacility } from '@ukef/dtfs2-common';
+import { FacilityClient, mongoDbClient, TfmFacilityClient } from './database-client';
+import { getSqlFacilityIds, getValidGefDealIdGenerator } from './helpers';
+import { aFacilityWithoutDealId } from './mock-data';
+
+const run = async () => {
+  const dataSource = await SqlDbDataSource.initialize();
+
+  await FacilityClient.init();
+  await TfmFacilityClient.init();
+
+  try {
+    const facilityIdsToInsert = await getSqlFacilityIds(dataSource);
+
+    const validGefDealIdGenerator = await getValidGefDealIdGenerator();
+
+    const facilities: Facility[] = facilityIdsToInsert.map((ukefFacilityId) => ({
+      ...aFacilityWithoutDealId(),
+      ukefFacilityId,
+      dealId: validGefDealIdGenerator(),
+    }));
+    await Promise.all(facilities.map((facility) => FacilityClient.insertIfNotExists(facility)));
+
+    const tfmFacilities: WithoutId<TfmFacility>[] = facilities.map((facilitySnapshot) => ({ facilitySnapshot }));
```

Comment:
Do you think we should add some with amendments?

Reply:
I considered it and eventually decided against as I assumed that, if people needed amendments, they could manually insert them. That seems like a bit of an unreasonable assumption now that I type it out, though, so I think I will randomly add in some amendments